src/plugins/chart_expressions/expression_partition_vis/public/utils/get_partition_theme.ts

```diff
@@ -137,18 +135,15 @@ export const getPartitionTheme: GetThemeFn = (
         }
       : null;
   const partition = {
-    fontFamily: chartTheme.barSeriesStyle?.displayValue?.fontFamily,
     outerSizeRatio: 1,
     minFontSize: 10,
     maxFontSize: 16,
```

Comment:
**nit:** can we type the intermediate variables?

```ts
const partition: PartialTheme['partition'] = {
  // ...properties
};
```

Reply:
done in https://github.com/elastic/kibana/pull/195269/commits/744642d426e46641eb1ec531c56848ce80f479c4
together with a couple of more cleanups